src/client/table.rs

```diff
@@ -355,45 +192,8 @@ impl From<ObTableBatchOperationResult> for Result<Vec<TableOpResult>> {
     }
 }
 
-struct ObTableStreamQuerier;
-
-impl ObTableStreamQuerier {
-    pub fn new() -> ObTableStreamQuerier {
-        ObTableStreamQuerier {}
-    }
-}
-
-impl StreamQuerier for ObTableStreamQuerier {
-    fn execute_query(
-        &self,
-        stream_result: &mut QueryStreamResult,
-        (part_id, ob_table): (i64, Arc<ObTable>),
-        payload: &mut ObTableQueryRequest,
-    ) -> Result<i64> {
-        let mut result = ObTableQueryResult::new();
-        ob_table.rpc_proxy.execute(payload, &mut result)?;
-        let row_count = result.row_count();
-        stream_result.cache_stream_next((part_id, ob_table), result);
-        Ok(row_count)
-    }
-
-    fn execute_stream(
-        &self,
-        stream_result: &mut QueryStreamResult,
-        (part_id, ob_table): (i64, Arc<ObTable>),
-        payload: &mut ObTableStreamRequest,
-    ) -> Result<i64> {
-        let mut result = ObTableQueryResult::new();
-        let is_stream_next = payload.is_stream_next();
-        ob_table.rpc_proxy.execute(payload, &mut result)?;
-        let row_count = result.row_count();
-        if is_stream_next {
-            stream_result.cache_stream_next((part_id, ob_table), result);
-        }
-        Ok(row_count)
-    }
-}
-
+#[allow(dead_code)]
+// impl ObTableStreamQuerier for obtable
```

Comment:
Can these traits including `ObTableStreamQuerier`, `Table` and `StreamQuerier` be removed? I remember that only just one implementation for them, and they are too complex to be mocked if we want to use them in unit tests.

Reply:
Fixed, only keep the ObTableClientStreamQuerier now, and rename it into StreamQuerier in case to keep the code clean.